src/commands/synthetics/__tests__/run-tests-lib.test.ts

```diff
@@ -390,6 +418,11 @@ describe('run-test', () => {
       const testStream = fs.createReadStream('/dev/null')
       jest.spyOn(fs, 'createReadStream').mockReturnValue(testStream)
 
+      const {AppUploadReporter} = jest.requireActual<typeof appUploadReporterModule>('../reporters/mobile/app-upload')
+      jest
+        .spyOn(appUploadReporterModule, 'AppUploadReporter')
+        .mockImplementation(() => new AppUploadReporter({stdout: {write: jest.fn()}} as any))
+
```

Comment:
Just out of curiosity why is this part needed?

Reply:
Sorry, didn't see the comment. It's unrelated - just to avoid having logs printed in the output by some reporters which aren't mocked